common.go

```diff
@@ -600,6 +601,9 @@ type Config struct {
 	// Allow short headers (experimental, and only applies to server).
 	AllowShortHeaders bool
 
+	// If true disables use of the Extended Master Secret extension
```

Comment:
`// DisableExtendedMasterSecret disables use of the Extended Master Secret extension.`

(The doc of `AllowShortHeaders` is wrong, but that's not a patch that will go upstream.)